packages/frontend/src/document/theoryselector.tsx

```diff
@@ -0,0 +1,83 @@
+import { For } from "solid-js";
+import type { TheoryLibrary } from "../stdlib/types";
+import type { ModelDocument } from "./types";
+import "./model_document_editor.css";
+
+interface TheorySelectorProps {
+    docHandle: any;
+    theories: TheoryLibrary;
+    doc: ModelDocument;
+}
+
+const TheorySelector = (props: TheorySelectorProps) => {
+    return (
+        <div id="input-selections" class="popup">
+            <h4 id="divisionCategoryHeader">Data and Knowledge</h4>
```

Comment:
The categories shouldn't be hard-coded. We can discuss alternative designs in person.